xml/Makefile.am

```diff
@@ -106,39 +106,63 @@ pacemaker-%.rng: $(RNG_files) best-match.sh Makefile.am
 	echo "<grammar xmlns='http://relaxng.org/ns/structure/1.0' datatypeLibrary='http://www.w3.org/2001/XMLSchema-datatypes'>" >> $@
 	echo "  <start>" >> $@
 	echo "    <element name='cib'>" >> $@
-	$(top_srcdir)/xml/best-match.sh cib $(*) $(@) "      "
+	./best-match.sh cib $(*) $(@) "      "
```

Comment:
On 09/02/16 17:03 -0800, Andrew Beekhof wrote:
>> @@ -106,39 +106,63 @@ pacemaker-%.rng: $(RNG_files) best-match.sh Makefile.am
>>  	echo "<grammar xmlns='http://relaxng.org/ns/structure/1.0' datatypeLibrary='http://www.w3.org/2001/XMLSchema-datatypes'>" >> $@
>>  	echo "  <start>" >> $@
>>  	echo "    <element name='cib'>" >> $@
>> -	$(top_srcdir)/xml/best-match.sh cib $(*) $(@) "      "
>> +	./best-match.sh cib $(*) $(@) "      "
> 
> I think that breaks when you build from outside of the source tree.
> Which people like to do apparently

Honestly, I cannot think of any use case that wouldn't break before
and is broken after this change.  In fact, the original usage was
even more fragile as you couldn't use Makefile* and best-match.sh
under any custom location, there would have to be a parental dir
hierarchy with a traversal containing "xml" directory at the very
least, which is not strictly needed now.

-- 
Jan (Poki)